src/LibLog/LibLog.cs

```diff
@@ -1823,7 +1824,7 @@ internal static class LogMessageFormatter
         /// <returns></returns>
         public static Func<string> SimulateStructuredLogging(Func<string> messageBuilder, object[] formatParameters)
         {
-            if(formatParameters == null)
+            if(formatParameters == null || !formatParameters.Any())
```

Comment:
Would `formatParameters.Length == 0` be better? (or would that be a "micro optimization"?)